be/src/Project/Web/Buyer/Buyer.Web/Areas/Products/Controllers/ProductController.cs

```diff
@@ -32,7 +36,8 @@ public async Task<IActionResult> Index(Guid? id)
                 Name = this.User.Identity.Name,
                 Token = await HttpContext.GetTokenAsync(ApiExtensionsConstants.TokenName),
                 BasketId = string.IsNullOrWhiteSpace(this.Request.Cookies[BasketConstants.BasketCookieName]) ? null : Guid.Parse(this.Request.Cookies[BasketConstants.BasketCookieName]),
-                ContentPageKey = "productPage"
+                ContentPageKey = "productPage",
+                SellerId = GuidHelper.ParseNullable((User.Identity as ClaimsIdentity).Claims.FirstOrDefault(x => x.Type == AccountConstants.Claims.OrganisationIdClaim)?.Value)
```

Comment:
**question:** Here too, why sellerId? If it is not needed anywhere, please remove it.